spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,169 @@
+require 'spec_helper'
+
+RSpec.describe FacilityFactory do 
+  before(:each) do
+    @colorado_facilities_data = DmvDataService.new.co_dmv_office_locations
+    @new_york_facilities_data = DmvDataService.new.ny_dmv_office_locations
+    @missouri_facilities_data = DmvDataService.new.mo_dmv_office_locations
+    @colorado = FacilityFactory.new
+    @new_york = FacilityFactory.new
+    @missouri = FacilityFactory.new
+    end
```

Comment:
Since all facility factories can process all state data, you could initialize only one factory to run your tests